.github/CODE_STYLE.md

```diff
@@ -9,19 +9,25 @@ Welcome to the code conventions section of Tegel, a comprehensive design system
  - Use meaningful variable and function names: Name your variables and functions in a way that clearly communicates their purpose.
 Avoid using single-letter variable names, instead of `i` for index, be descriptive and name the variable `index`.
  - Use single quotes (') for strings.
+ - If applicable, add unit test. 
 
 ## Branching
 - Branch out of `develop` branch for day-to-day work
 - Branch named `main` is updated once release is done by merging develop to it
 - Use task type as to separate branches in different categories (fix, feat, chore, build, docs, release...)
 - Start the name of the branch with Jira ticket number followed by short description of task
 - example: `fix/CDEP-1369-banner-color-correction`
+- If Jira ticket number is not available to you, use Github ticket number as prefix instead
```

Comment:
Maybe should clarify that if a github issue doesn't exist it should be created before creating a branch.

Reply:
Good point!